assets/installer.py

```diff
@@ -69,7 +71,17 @@ def get_confirmation():
             print("Aborted!")
             sys.exit()
 
-
+            
+def is_ensurepip_pip_version_ok():
+    try:
+        import ensurepip
```

Comment:
Why are the imports inside the try/except?

Reply:
I'm not sure if the import has side effects, or if ensurepip will always be present. I didn't want it to influence the installation on Linux or Mac.

Reply:
`ensurepip` is only present in Python 3.4 and onwards. And according to a comment further down it seems like Debian removes the `ensurepip` module. Since Lektor 3.2 still supports 2.7 I think the try catch here is fine.

Reply:
👍